### DISCUSSION TITLE
Why is the F-Droid release so out of date?

### DISCUSSION THREAD

**Discussion by JimmyCushnie** [kiwix/kiwix-android#3517] [Q&A] [answered]
Hello,

According to the [releases page of this repo](https://github.com/kiwix/kiwix-android/releases), the latest kiwix release is 3.7.1, from 2023-06-07. Furthermore, 3.7.1 is the version you get if you press "direct APK" from [the the kiwix.org download page](https://kiwix.org/en/applications/).

However, the latest version on [the F-Droid page](https://f-droid.org/en/packages/org.kiwix.kiwixmobile/) is 2.4, from 2018-09-21. Why is F-Droid so out of date? Could somebody please explain this?

(For completionism's sake, the [play store version](https://play.google.com/store/apps/details?id=org.kiwix.kiwixmobile) hasn't had an update since 2021-10-22, so it must be out of date as well.)

**Comment by s-i-m-s** [comment, marked as answer]
IIUC the deal with fdroid is kiwix is using some library that isn't FOSS so isn't allowed on fdroid until they switch away from the offending library which has been being worked on slowly for years now.

IIUC the deal with google play has to do with not complying with how google wants apps to interact with the file system, from the release notes I _think_ that's been resolved with the latest 3.8.0 release but it hasn't been approved by google yet.

My suggestion was to direct people to the up to date copies at least until the other locations can be updated as the old versions of kiwix are either partly or completely broken on newer android versions.

Like right now you go to https://kiwix.org/en/applications/ and the first two options are both out of date.

I asked about this almost a year ago https://www.reddit.com/r/Kiwix/comments/119lwjl/why_isnt_the_play_store_up_to_date/ and while there have been several new releases since then the store situations haven't really changed.

**Comment by kelson42** [comment]
We are working to bring back Kiwix to FDroid, but this is going to take a bit of work and additional months

**Reply by timkrief** [reply]
Thanks for your work

**Reply by kelson42** [reply]
This is a matter of weeks from now, actually at the very top of our todo list.
